frontend/src/components/SeedlotRegistrationSteps/InterimStep/index.tsx

```diff
@@ -135,122 +97,69 @@ const InterimStorage = ({ setStep }: InterimStorageStepProps) => {
     setValidationObj(newValidObj);
   };
 
-  const getMinDate = () => {
-    let minDate = '';
-    if ((interimForm.startDate !== '' && interimForm.endDate === '')
-      || (interimForm.startDate !== '' && interimForm.endDate !== '')) {
-      minDate = interimForm.startDate;
-    } else if ((interimForm.startDate === '' && interimForm.endDate === '')
-      || (interimForm.startDate === '' && interimForm.endDate !== '')) {
-      return minDate;
-    }
-    return minDate;
-  };
-
-  const getTodayDate = () => {
-    const today = new Date();
-    const dd = String(today.getDate()).padStart(2, '0');
-    const mm = String(today.getMonth() + 1).padStart(2, '0'); // January is 0!
-    const yyyy = today.getFullYear();
-    const stringDate = `${yyyy}/${mm}/${dd}`;
-    return stringDate;
-  };
-
-  const getMaxDate = () => {
-    let maxDate = '';
-    if ((interimForm.startDate === '' && interimForm.endDate === '')
-      || (interimForm.startDate !== '' && interimForm.endDate === '')) {
-      maxDate = getTodayDate();
-    } else if ((interimForm.startDate === '' && interimForm.endDate !== '')
-      || (interimForm.startDate !== '' && interimForm.endDate !== '')) {
-      maxDate = interimForm.endDate;
-    }
-    return maxDate;
-  };
-
```

Comment:
Why remove the max and min dates of the date inputs?

Reply:
In the collab meeting we had I think we decided that the only rule for the dates is: End date has to be after Start date. 
This rule can be enforced simply by providing the End Date input a prop `minDate` with a value of the start date: 
`minDate={state.startDate}`. So those functions are not needed.

Reply:
Forgot about that discussion, sorry. 😆